src/component/WeatherCard.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const WeatherCard = () => (
+  <div>WeatherCard</div>
+);
+
+export default WeatherCard;
```

Comment:
- [ ] Again here consider writing a unit test for this component as per requirement.

Example
```
import React from 'react';
import { render } from '@testing-library/react';
import WeatherCard from './WeatherCard';

test('renders the WeatherCard component', () => {
  const { getByText } = render(<WeatherCard />);
  
  // Assert that the text "WeatherCard" is present in the rendered component.
  const weatherCardElement = getByText('WeatherCard');
  expect(weatherCardElement).toBeInTheDocument();
});
```